requre/cassette.py

```diff
@@ -304,8 +304,9 @@ def create_from_dict_with_list(cls, dict_repr: dict, cassette: Any):
     def create_from_list(list_repr: list, cassette: Any):
         if len(list_repr) == 0:
             raise StorageNoResponseLeft(
-                "No responses left. Try to regenerate response file "
-                f"({cassette.storage_file})."
+                f"Unable to find response in file {cassette.storage_file}. "
+                "You are in read-only mode now, that's why you are seeing this error message. "
+                "You should record the test case again."
```

Comment:
What about mention also how to do the re-recording?

e.g. mention removing the file and rerunning the tests

Reply:
Could be. I've used text what were suggested in issue and just little bit tweaked it.

Reply:
e.g. When I guided Ben (or other newcomers) through the test regeneration in OGR, they don't know this.

```suggestion
                f"Unable to find response in file {cassette.storage_file}. "
                "You are in read-only mode now, that's why you are seeing this error message. "
                "You should record the test case again by removing the response file and running the test(s) again."
```

Reply:
Yep, I'll used similar text and little bit improved it. thank you very much.